assets/blocks/managementBook.block

```diff
@@ -0,0 +1,9 @@
+{
+    "shadowCasting" : false,
+    "luminance" : 10,
+    "hardness" : 10,
+    "penetrable" : true,
+    "entity" : {
+        "prefab" : "masteroforeon:managementBook"
```

Comment:
Not line-relevant, but for this .block is there actually a texture? You could perhaps use a similar approach as the research book where you've got a pedestal + a an item sort of hovering on it. There's also a neato pedestal in Light & Shadow, although it may still be stuck in a branch.